config/credentials.yml.enc

```diff
@@ -1 +1 @@
-a0HWBU0Q5ZDoV4+RdAK5UFa9t+2CeiAmUvZYnd7PcNlSVrMDXH3ltjLBM5m9NgsREZmoyG4giJe1U+I3pq+NohlVl5d0H9PpQ2AiUPDD9ykQNylDaU16A70+TcslpVMmHJL7G4mwy+lU186pBJ482TKj8RR9L9g1wIKjy1GtKR84sudtiNcJQ0FNWqZ6Ett17jYB+F9qsBAcBDNWelUvEP9P0yOPTDxhWOhG4MaVbT3ejVw39k9OqX1e1IXMn6ias2ui30qOcnj1ssxeLlCcqimoOu5Ve8Gg+1mcRdOjwiR68Sli6CxCdk2BNCo9wfavVjpnCyEpKr2Wb/zhYYXNTzycUH4elfZ9McBOi0l+9GOKOtqoQS75MxBVlIFh9eh7HmatxV8uaeXj56Z2gf7YGbNV9ijNYS0zvnUO--4E6Rham/5eCiEFFZ--0Djqff/c3KFXDsunPKJVSQ==
\ No newline at end of file
+M4iZD8NDL07GXbjXu+IiSgjdjTvYr7fNBHH5oisR9lWf9JbUf3uDqPJ1KK8wVvHW7PWsqIxvw5rGgl7doJ0ATEW94Sr7GSQbaxmL07AoGby8us97BafSgBXFAHx0er7Dq6MT8kh4P7MmTRx4BmLqzdUdbBfWguvS2RYpTl64yyzc8j1IoArIwh0pD+BwKtdXAQJ5wbQPM4ZsFr0ZZQrddLmE/mW1dY/oUmtbi5IW8D9+UaZTa5ty5LK3SIjER+07sLS8or89SgIXCnjPLhr1bOcu7aNsu32pfkuZGPNXnUqRUwGlbKjJ22HULzSwdwCa5vIC0di46djIqKjOnjp6DO+EZOBD1YY+US3fWHmuC0DyFulJI/DoD8QIsBkopbZp5/ngwYVQRznpm+ZZ+yAQAmX4vAIV--mkhbqaZAybTDudHJ--PErwpyZvIVis1+jzuk5lsA==
```

Comment:
You can add a file like this to your `.gitignore` so that you don't share your credentials on GH